README.md

```diff
@@ -201,7 +201,7 @@ config :plug, :mimes, %{
 ```
 
 
-And then re-compile plug: (per: http://hexdocs.pm/plug/Plug.MIME.html)
+And then re-compile plug: (per: https://hexdocs.pm/plug/1.1.3/Plug.MIME.html)
```

Comment:
I wonder if this should link to the [mime project](https://hexdocs.pm/mime/MIME.html) docs instead since these docs were [removed in 1.2.0](https://github.com/elixir-lang/plug/commit/03e5efb5b957ecc250455ecaf6bc615187e21234#diff-eefdb392f58102aed281d3c5f08ad377R2).